src/sensors/bno055sensor.cpp

```diff
@@ -39,6 +43,11 @@ void BNO055Sensor::motionSetup() {
     imu.setAxisRemap(Adafruit_BNO055::REMAP_CONFIG_P0);
     imu.setAxisSign(Adafruit_BNO055::REMAP_SIGN_P0);
     m_Logger.info("Connected to BNO055 at address 0x%02x", addr);
+
+    //Adafruit BNO055's use external crystal. Enable it, otherwise it does not work.
+    delay(1000);
```

Comment:
Why is there a delay of 1000? I didn't notice that we need it during testing.

Reply:
Removed the double delay. But the 1000ms delay should still be present to configure the external crystal (which is like ~600ms).